docker-cloud/terraform/sg.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_security_group" "lb" {
+  name   = "docker-cloud"
+  vpc_id = data.aws_vpc.default.id
+
+  ingress {
+    protocol    = "tcp"
+    from_port   = 80
+    to_port     = 80
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
```

Comment:
it's not wrong to have this but SGs generally default allow all egress so I don't think you actually need it

Reply:
You are right, thanks!